src/community/gwc-sqlite/src/main/java/org/geoserver/gwc/web/blob/SqliteMultipartFilter.java

```diff
@@ -0,0 +1,22 @@
+package org.geoserver.gwc.web.blob;
+
+import org.geoserver.filters.GeoServerFilter;
+import org.springframework.web.multipart.support.MultipartFilter;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class SqliteMultipartFilter extends MultipartFilter implements GeoServerFilter {
+    @Override
+    protected void doFilterInternal(HttpServletRequest request, HttpServletResponse response, FilterChain filterChain) throws ServletException, IOException {
+        String path = ((HttpServletRequest) request).getRequestURI();
+        if (path.contains("/sqlite/")) {
```

Comment:
Ok I have investigate this and yes this is just for the integration. I have manually tested with path ``gwc/rest/sqlite/`` and it works has expected.